handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
 };
 
 Character.prototype = Object.create(Player.prototype);
 Character.prototype.constructor = Character;
 
+Character.prototype.addItem = function(newItem) {
+    for (var item in this.inventory) {
+        if (this.inventory[item].name === newItem.name) {
+            this.inventory[item].quantity += newItem.quantity;
+            return "Added new quantity to existing quantity.";
```

Comment:
I've got an enum for inventory results, but I'm a bit unsure how to handle your last comment here. 

My assumption was that, for example, all items with the name "Shortsword" have the same values, and that variants such as "Battered Shortsword" have separate values for price and weight and such, and can be considered entirely different items.

But let's say I pursue that, and separate items from their ideal price. I'm having difficulty picturing where related values can be properly stored. I might have suggested a key-value pairing, where an item is the key, and the price is the value, but with what we are going with above, I can't do that.